modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -547,12 +558,91 @@ SHOW USERS
 |false
 |false
 |<null>
+|"jake"
+|["PUBLIC"]
+|false
+|false
+|<null>
+5+a|Rows: 2
+|===
 
-5+a|Rows: 1
+To inspect users' xref:authentication-authorization/auth-providers.adoc[Auth Providers], use `SHOW USERS WITH AUTH`. This command will produce a row per user per Auth Provider and will yield the following two columns in addition to those output by `SHOW USERS`:
```

Comment:
I think it might be better to have a sub-section of Listing Users which is Listing Users With Auth or something, that way we can refer to it directly and it is less likely to just disappear among the other things

(cause at the moment its not very visible what is examples/example descriptions/example results/tables of return columns for the command/info on behaviour)

Reply:
If you want I could take a stab at restructuring this bit to be more readable/clear distinctions

Reply:
done